src/components/number/number.md

```diff
@@ -0,0 +1,39 @@
+```js
```

Comment:
I'm trying out a new pattern to make our examples a bit more consistent.

Can you follow this guide https://github.com/mavenlink/design-system/blob/master/docs/documentation.md ?